source/developers/tutorial.markdown

```diff
@@ -0,0 +1,140 @@
+---
+layout: page
+title: "Development tutorial"
```

Comment:
The name is too generic. It should mention what the end result is of the tutorial.